src/server/game/Texts/ChatTextBuilder.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2016+ AzerothCore <www.azerothcore.org>, released under GNU AGPL v3 license, you may redistribute it and/or modify it under version 2 of the License, or (at your option), any later version.
+ * Copyright (C) 2016+ AzerothCore <www.azerothcore.org>, released under GNU AGPL v3 license, you may redistribute it and/or modify it under version 3 of the License, or (at your option), any later version.
```

Comment:
that's not what I meant lol, see other examples of AGPL 3 header

Reply:
Just use other headers

Reply:
I tried at least xD will do whenever I get time